docs/product-user-guide/src/main/asciidoc/decision-tables-rule-table-entries-ref.adoc

```diff
@@ -0,0 +1,85 @@
++
+To expand a text according to the pattern `forall(_DELIMITER_){_SNIPPET_}`, repeat the `_SNIPPET_` once for each of the values of the comma-separated list in each of the cells below, insert the value in place of the symbol `$`, and join these expansions by the given `_DELIMITER_`. Note that the `forall` construct may be surrounded by other text.
++
+If the first cell contains an object, the completed code snippet is added to the conditional element from that cell. A pair of parentheses is provided automatically, as well as a separating comma if multiple constraints are added to a pattern in a merged cell. If the first cell is empty, the code snippet in this cell must result in a valid conditional element on its own. For example, instead of `Person` as the object type and `age` as the constraint (separate cells), you can enter `Person( age == $1)` in the constraint cell and leave the cell above it empty.
+* *Third cell:* Text in the third cell below `CONDITION` is a descriptive label that you define for the column, as a visual aid.
+* *Fourth cell:* From the fourth row on, non-blank entries provide data for interpolation. A blank cell omits the condition or constraint for this rule.
+
+Actions::
+
+For columns headed `ACTION`, the cells in successive lines result in an action statement:
+
+* *First cell:* Text in the first cell below `ACTION` is optional. If present, the text is interpreted as an object reference.
+* *Second cell:* Text in the second cell below `ACTION` is processed as a constraint on the object reference in the first cell, if present. The code snippet in this cell is modified by interpolating values from cells farther down in the column. For a singular insertion, mark the position for including the contents of a cell with the symbol `$param`. Multiple insertions are possible if you use the symbols `$1`, `$2`, etc., and a comma-separated list of values in the cells below. However, do not separate `$1`, `$2`, etc., by commas, or the table will fail to process.
++
+A text without any marker symbols can execute a method call without interpolation. In this case, use any non-blank entry in a row below the cell to include the statement. The `forall` construct is supported.
++
+If the first cell contains an object, then the cell text (followed by a period), the text in the second cell, and a terminating semicolon are strung together, resulting in a method call that is added as an action statement for the consequence. If the first cell is empty, the code snippet in this cell must result in a valid action element on its own.
+* *Third cell:* Text in the third cell below `ACTION` is a descriptive label that you define for the column, as a visual aid.
+* *Fourth cell:* From the fourth row on, non-blank entries provide data for interpolation. A blank cell omits the condition or constraint for this rule.
+
+Metadata::
+
+For columns headed `METADATA`, the cells in successive lines result in a metadata annotation for the generated rules:
```

Comment:
"successive" is another that I pause on for translation potential issues. Would "consecutive" be better?